src/content/blog/2022-08-05-new-config-system-part-2.md

```diff
@@ -0,0 +1,512 @@
+---
+layout: post
+title: "ESLint's new config system, Part 2: Introduction to flat config"
+teaser: "ESLint's new config system, nicknamed flat config, is designed to be both familiar and much simpler than the original config system."
+tags:
+  - Configuration
+  - Command Line
+authors:
+  - nzakas
+categories:
+  - API Changes
+---
+
+In [my previous post](https://eslint.org/blog/2022/08/new-config-system-part-1/), I talked about how the eslintrc config system had grown to be more complex than necessary through a series of small, incremental changes. The flat config system, on the other hand, was designed from the start to be simpler in a number of ways. We took all of the learnings from the previous six years of ESLint development to come up with a holistic approach to configuration that took the best of eslintrc and combined it with the way other JavaScript-related tools handled configuration. The result is something that hopefully feels familiar to existing ESLint users and is far more powerful than what was possible before.
```

Comment:
```suggestion
In [my previous post](https://eslint.org/blog/2022/08/new-config-system-part-1/), I talked about how the eslintrc config system had grown to be more complex than necessary through a series of small, incremental changes. The flat config system, on the other hand, was designed from the start to be simple in a number of ways. We took all of the learnings from the previous six years of ESLint development to come up with a holistic approach to configuration that took the best of eslintrc and combined it with the way other JavaScript-related tools handled configuration. The result is something that hopefully feels familiar to existing ESLint users and is far more powerful than what was possible before.
```
NIT

Reply:
Simpler is correct here because it is in comparison to the old config system.